test/unit/remote-config/remote-config-api-client.spec.ts

```diff
@@ -344,7 +344,8 @@ describe('RemoteConfigApiClient', () => {
         stubs.push(stub);
         const expected = new FirebaseRemoteConfigError('failed-precondition', message);
         return apiClient.validateTemplate(REMOTE_CONFIG_TEMPLATE)
-          .should.eventually.be.rejected.and.deep.equal(expected);
+          .should.eventually.be.rejectedWith(expected.message)
```

Comment:
It looks like this changes what's being tested - Why are we expecting a message to be thrown now, rather than an Error?

Reply:
`rejectedWith()` and `throws()` always test for error messages (or string patterns) only. Given that Chai can no longer compare `Error` objects (see the description of the PR), we have to assert on message and code properties separately now.

Reply:
Just curious. Do you think we can use `.should.eventually.be.rejected.and.deep.include(error)` here?

Reply:
Yes, that actually works since `expected` does not contain a stack trace. Updated this file.